bin/faucet/.env

```diff
@@ -15,3 +15,4 @@ MIDEN_FAUCET_POW_BASELINE=
 MIDEN_FAUCET_API_KEYS=
 MIDEN_FAUCET_ENABLE_OTEL=false
 MIDEN_FAUCET_STORE=
+MIDEN_FAUCET_EXPLORER_URL=https://testnet.midenscan.com/
```

Comment:
I think I'd leave this with no default, otherwise it seems like it would be very easy to have an incorrect setting